bin/disco_ssh.py

```diff
@@ -133,16 +135,31 @@ def build_ssh_cmd(self, ips):
         """
         Given a list of ip addresses, build an ssh command to tunnel through n-1 ips to reach the n-th ip
         """
-        command = " ".join([
-            "ssh -At {} {}".format(SSH_OPTIONS, ip)
-            for ip in ips])
+        if self.tunnel:
+            lport, rport = self.tunnel.split(":")
+            host = self.args["<host>"]
```

Comment:
Yes, that's a limitation of the code as it stands.  I would call it a constraint, not an ambiguity, but maybe I don't understand exactly what you are referring to.